kong/init.lua

```diff
@@ -892,6 +892,12 @@ function Kong.init_worker()
     return
   end
 
+  if kong.configuration.role == "control_plane" then
+    if kong.clustering then
+      kong.clustering:init_worker()
```

Comment:
Perhaps we should check line 999 and change it:
```
    if not kong.sync then
      kong.clustering:init_worker()
    end
```

Reply:
> Perhaps we should check line 999 and change it:
> 
> ```
>     if not kong.sync then
>       kong.clustering:init_worker()
>     end
> ```

No! We need start v1 for CP, that's all this PR is about

Reply:
maybe:
```
    if not kong.sync or is_control_plane(kong.configuration) then
      kong.clustering:init_worker()
    end
```

Reply:
> maybe:
> 
> ```
>     if not kong.sync or is_control_plane(kong.configuration) then
>       kong.clustering:init_worker()
>     end
> ```

It's not the same. DP needs to exit the handler after the initialization.